multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -163,6 +142,7 @@ func (r *MemberClusterAnnounceReconciler) Reconcile(ctx context.Context, req ctr
 		klog.ErrorS(err, "Failed to add member cluster to ClusterSet", "cluster", memberAnnounce.ClusterID)
 		return ctrl.Result{}, err
 	}
+	r.addMemberStatus(memberID, v1.ConditionTrue)
```

Comment:
Should we move it to the end of the func after adding the Finalizer?

Reply:
I think it's better to place it before adding Finalizer since member is actually connected and able to do export/import etc.